### PR TITLE
 [DOC] - Find good images for examples docs [documentation] [good first issue] #1568 

### PR DESCRIPTION
**Pull request by Abhash297**
#### Reference Issues/PRs

Gathered relevant images for the topics.
Added a few references where it was necessary. 

No new dependency

##### For all contributions
- [x] I've added myself to the [list of contributors](https://github.com/aeon-toolkit/aeon/blob/main/.all-contributorsrc). Alternatively, you can use the [@all-contributors](https://allcontributors.org/docs/en/bot/usage) bot to do this for you.
- [x] The PR title starts with either [ENH], [MNT], [DOC], [BUG], [REF], [DEP] or [GOV] indicating whether the PR topic is related to enhancement, maintenance, documentation, bugs, refactoring, deprecation or governance.

##### For new estimators and functions
- [ ] I've added the estimator to the online [API documentation](https://www.aeon-toolkit.org/en/latest/api_reference.html).
- [ ] (OPTIONAL) I've added myself as a `__maintainer__` at the top of relevant files and want to be contacted regarding its maintenance. Unmaintained files may be removed. This is for the full file, and you should not add yourself if you are just making minor changes or do not want to help maintain its contents.

##### For developers with write access
- [ ] (OPTIONAL) I've updated aeon's [CODEOWNERS](https://github.com/aeon-toolkit/aeon/blob/main/CODEOWNERS) to receive notifications about future changes to these files.

### CODE REVIEW

**Comment by TonyBagnall** [comment]
[@all-contributors](https://allcontributors.org/docs/en/bot/usage) add @Abhash297 for doc

**Comment by Abhash297** [comment]
Please review and merge

**Comment by TonyBagnall** [comment]
so I appreciate the effort. Can you confirm that all these images are freely available not under copywrite?

**Comment by Abhash297** [comment]
The images I took from research papers have been added to the reference section in the subsequent pages. Some Images like the one in "pre processing" were taken from an article but it was a generic image so I did not add it to the reference.

File: docs/examples.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Examples
+# Example
```
Comment (TonyBagnall) [inline docs/examples.md:1]: I think it should remain Examples

**Review by TonyBagnall — CHANGES_REQUESTED** [review]
lgtm, one small change

File: docs/examples.md (hunk 1 above)
Comment (Abhash297) [reply docs/examples.md]: Yes, change made to examples.md

**Review by TonyBagnall — DISMISSED** [review]

**Comment by MatthewMiddlehurst** [comment]
IMO we should cut this back a bit, select a few examples/images and just focus on this in this PR. If the image is in `examples/*/img`, it should be used in the notebook itself.

**Comment by MatthewMiddlehurst** [comment]
`examples/benchmarking/img/benchmark_algo1.png`
`examples/classification/img/early.png`
`examples/clustering/img/partition_tscl.png`
`examples/datasets/img/example_ds.png`
`examples/datasets/img/load_data.png`
`examples/transformations/img/catch22.png`
`examples/transformations/img/mini_rocket.png`

These look fine, but some could use more informative names. Would select a couple from these for the first version of this PR (you can keep all of them in this one or a separate PR if you wish).

**Review by MatthewMiddlehurst — COMMENTED** [review]
There is an extra `typecasting.png` is the `docs/` directory

**Pull request closed without merging**